docs/source/guide/frontend_reference.md

```diff
@@ -307,10 +319,13 @@ Annotation is updated.
 
 Annotation is selected.
 
-| Argument      | Type     | Description              |
-| ------------- | -------- | ------------------------ |
-| `labelStudio` | `Object` | Instance of Label Studio |
-| `annotation`  | `Object` | Current annotation       |
+| Argument              | Type      | Description                                   |
+|-----------------------|-----------|-----------------------------------------------|
+| `annotation`          | `Object`  | Current annotation                            |
+| `previousAnnotation`  | `Object`  | Previous annotation                           |
+| `payload`             | `Object?` | Additional information                        |
```

Comment:
Check question marke - and could we just say "Payload content" or something under Description